dashboard/src/components/util/RenderCounter.js

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+
+class RenderCounter extends React.Component {
```

Comment:
is this being used?

Reply:
Not currently, but I want to keep it around as a debugging tool.

Reply:
It will print a running count of component updates and is useful for confirming that unnecessary renders aren't getting triggered as we work on different components.